.gitignore

```diff
@@ -64,3 +64,7 @@ doc
 codalab/tests
 codalab/tags
 notes.otl
+
+# Distribution / packaging
+.Python
+env/
```

Comment:
:sheep: can you add a newline here? (we try to have a newline at the end of all of our files, which you can configure your IDE to do automatically for you!)